js/packages/web/locale/ja.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Address to mint edition to": "",
+  "Buy exclusive {name} NFTs.": "専用の{name} NFTを購入する。",
+  "Congratulations": "おめでとうございます",
+  "Got it": "",
+  "Mint": "発行",
+  "Mint to": "発行先",
+  "My Items": "マイアイテム",
+  "New editions have been minted please view your NFTs in": "",
+  "Number of editions to mint": "発行数",
+  "The amazing world of {name}": "ようこそ{name}の世界へ",
+  "Total cost": "トータルコスト"
+}
```

Comment:
```suggestion
}

```

Reply:
and here :D i just left the suggestions in case you want to go for the change

Reply:
but i dont think there is really a hard technical reason for it, other than convention i.e check this out https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I recreated the json by lingui and it added a newline at the end.